python/llm/README.md

```diff
@@ -67,6 +67,7 @@ Over 20 models have been optimized/verified on `bigdl-llm`, including *LLaMA/LLa
 | CodeLlama  | [link](example/CPU/HF-Transformers-AutoModels/Model/codellama) | [link](example/GPU/HF-Transformers-AutoModels/Model/codellama)  |
 | Skywork    | [link](example/CPU/HF-Transformers-AutoModels/Model/skywork)                 |    |
 | InternLM-XComposer    | [link](example/CPU/HF-Transformers-AutoModels/Model/internlm-xcomposer)   |   |
+| WizardCoder | [link](example/CPU/HF-Transformers-AutoModels/Model/wizardcoder) | |
```

Comment:
WizardCoder-Python

Reply:
WizardCoder-Python and WizardCoder are based on different backbones. So make it clear.

Reply:
maybe change the folder to wizardcoder-python as well